src/services/comments.js

```diff
@@ -70,23 +78,23 @@ module.exports = class ProjectsHelper {
 	}
 
 	/**
-	 *  comment list
+	 * Comment list
 	 * @method
 	 * @name list
 	 * @param {Object} req - request data.
 	 * @returns {JSON} - comment list
 	 */
-	static async list(resource_id, organization_id) {
+	static async list(resource_id, page_value = '', context = '', loggedInUserId, organization_id) {
```

Comment:
@priyanka-TL We should just call it userId in the argument. Also update the function signature with correct description and covering all arguments.